R2API.RecalculateStats/README.md

```diff
@@ -29,5 +29,8 @@ These stat changes are represented in the StatHookEventArgs, which includes argu
 
 ## Changelog
 
+### '1.0.1'
```

Comment:
Version bump should ideally be a minor instead of a patch,since we are adding new api surface.